opensearch/src/main/java/org/opensearch/sql/opensearch/response/OpenSearchResponse.java

```diff
@@ -92,23 +95,34 @@ public Iterator<ExprValue> iterator() {
         return (ExprValue) ExprTupleValue.fromExprValueMap(builder.build());
       }).iterator();
     } else {
+      float maxScore = hits.getMaxScore();
```

Comment:
nit: could create variable for `hit.getScore()` to stay consistent with the checks on line 11

Reply:
sorry - I don't understand.  Line 11 is 
```
import java.util.Arrays;
```

Reply:
But you make a decent point of saving as a `ExprFloatValue | null`.

Reply:
updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/228/commits/3ba3c855e02d540254da360608fc143de2e27707

Reply:
Oh.  We don't want that.  `maxscore` is the same for each row, while `score` can be different.